src/main/java/com/ita/if103java/ims/dao/impl/AssociateDaoImpl.java

```diff
@@ -72,6 +72,16 @@ public Associate findById(Long accountId, Long id) {
         }
     }
 
+    @Override
+    public List<Associate> getAssociates(String sort, int size, long offset, long accountId) {
+        try {
+            final String query = " select * from associates where account_id=? order by " + sort + " limit ? offset ?";
```

Comment:
Please move query to the constants below and use String.format to replace sorting